crates/optimism/node/src/node.rs

```diff
@@ -519,6 +544,9 @@ where
         let Self { pool_config_overrides, .. } = self;
         let data_dir = ctx.config().datadir();
         let blob_store = DiskFileBlobStore::open(data_dir.blobstore(), Default::default())?;
+        // supervisor used for interop
+        let supervisor_client =
+            SupervisorClient::new(self.supervisor_http.clone(), self.supervisor_safety_level).await;
```

Comment:
it was correct before! this is the right place to check if supervisor url is set if interop is active because this is where we have access to the chainspec.

Reply:
same with safety level. we can even combine those two, supervisor url and safety level, into one new config type `InteropConfig` and have `with_supervisor` take that as param instead.

Reply:
But if we have a default url then we won't need to check 🤔

Reply:
true, nvm. let's add an info log message if the default url is used, smthg like "no supervisor endpoint was provided in cli, using default url {DEFAULT_URL}"

Reply:
or at least log the url that is being used, maybe that's enough